src/interpreter/ByteCode.h

```diff
@@ -1385,9 +1236,13 @@ class Throw : public ByteCode {
         , m_functionType(functionType)
 #endif
     {
+        ASSERT(getBaseSize() == (sizeof(Throw)));
     }
 
-    uint32_t tagIndex() const { return m_tagIndex; }
+    uint32_t tagIndex() const
```

Comment:
Why this is changed?

Reply:
It was a whitespace mistake on my part and the auto-formatter edited it. It should be reverted now.